File: app/src/main/java/com/casic/titan/demo/activity/DemoPagingActivity.java
package com.casic.titan.demo.activity;

import android.os.Build;
import android.os.Bundle;

import com.casic.titan.demo.R;
import com.casic.titan.demo.bean.UseCase;
import com.casic.titan.demo.databinding.ActivityDemoPagingBinding;

import dagger.hilt.android.AndroidEntryPoint;
import pers.fz.mvvm.base.BaseActivity;
import pers.fz.mvvm.util.common.DateUtil;
import pers.fz.mvvm.util.common.ThreadExecutor;
import pers.fz.mvvm.util.log.LogUtil;
import pers.fz.mvvm.viewmodel.MainViewModel;

@AndroidEntryPoint
public class DemoPagingActivity extends BaseActivity<MainViewModel, ActivityDemoPagingBinding> {
    private UseCase useCase;

    @Override
    protected int getLayoutId() {
        return R.layout.activity_demo_paging;
    }

    @Override
    public String setTitleBar() {
        return null;
    }

    @Override
    public void initView(Bundle savedInstanceState) {
    }

    @Override
    public void initData(Bundle bundle) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
            useCase = bundle.getParcelable("args", UseCase.class);
        } else {
            useCase = bundle.getParcelable("args");
        }
        toolbarBind.getToolbarConfig().setTitle(useCase.getName());

    }
}

File: app/src/main/java/com/casic/titan/demo/activity/DemoPagingActivity.kt
package com.casic.titan.demo.activity

import android.os.Build
import android.os.Bundle
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.NavHostFragment
import com.casic.titan.demo.R
import com.casic.titan.demo.bean.UseCase
import com.casic.titan.demo.databinding.ActivityDemoPagingBinding
import com.casic.titan.demo.fragment.DemoPagingFragmentDirections
import dagger.hilt.android.AndroidEntryPoint
import pers.fz.mvvm.base.BaseActivity
import pers.fz.mvvm.viewmodel.EmptyViewModel

@AndroidEntryPoint
class DemoPagingActivity : BaseActivity<EmptyViewModel?, ActivityDemoPagingBinding?>() {
    private var useCase: UseCase? = null

    override fun getLayoutId(): Int {
        return R.layout.activity_demo_paging
    }

    override fun setTitleBar(): String? {
        return null
    }

    override fun initView(savedInstanceState: Bundle?) {
    }

    override fun initData(bundle: Bundle) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
            useCase = bundle.getParcelable("args", UseCase::class.java)
        } else {
            useCase = bundle.getParcelable("args")
        }
        toolbarBind.toolbarConfig?.title = useCase?.name
    }
}